clippy.json

```diff
@@ -63,5 +63,11 @@
     "mail.google.com": [
         "Do you need glasses?",
         "Maybe try Outlook? wink wink..."
+    ],
+    "vk.com": [
+        "Дуров, верни стену.",
+        "При помощи фена и простой ванны, можно сделать отличное джакуззи.."
```

Comment:
No worries :)

Each line of JSON file should end with `,` (comma), except for the last line. You can use https://jsonlint.com to validate contents of `clippy.json` before submitting again.

Reply:
Oke. :)